apps/ledger-live-mobile/package.json

```diff
@@ -68,8 +68,9 @@
     "download-hermes-profile": "zx ./scripts/download-hermes-profile.mjs",
     "e2e:loadConfig": "ts-node ./e2e/bridge/start-server.ts",
     "check": "pnpm lint; pnpm typecheck",
+    "assets:replace": "zx ./scripts/replace-assets.mjs",
     "unimported": "unimported",
-    "assets:replace": "zx ./scripts/replace-assets.mjs"
+    "knip-check": "pnpm knip --directory ../.. --workspace apps/ledger-live-mobile"
```

Comment:
[ASK] Same as desktop, do you really need to specify the workspace since you are already in it